Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -63,6 +76,8 @@
 import org.springframework.core.env.Environment;
 import org.springframework.stereotype.Component;
 
+import com.db4o.query.Query;
```

Comment:
Pretty sure that's the wrong query class

Reply:
Yea. Never used that db4 query class, imported by mistake and it was never used. Have used org.apache.lucene.search.Query. Will remove the unused import.